lightning/src/ln/reorg_tests.rs

```diff
@@ -281,15 +282,25 @@ fn do_test_unconf_chan(reload_node: bool, reorg_after_reload: bool, use_funding_
 	if !reorg_after_reload {
 		if use_funding_unconfirmed {
 			let relevant_txids = nodes[0].node.get_relevant_txids();
-			assert_eq!(&relevant_txids[..], &[chan.3.txid()]);
-			nodes[0].node.transaction_unconfirmed(&relevant_txids[0]);
+			assert_eq!(relevant_txids.len(), 1);
+			let block_hash_opt = relevant_txids[0].1;
+			assert!(block_hash_opt.is_some());
+			let expected_hash = nodes[0].get_block_header(chan_conf_height).block_hash();
+			assert_eq!(block_hash_opt.unwrap(), expected_hash);
```

Comment:
FWIW, can do in one assert by wrapping `expected_hash` in `Some`.

Reply:
Done and squashed in latest commit.